src/src/AdminBundle/Resources/views/base.html.twig

```diff
@@ -7,59 +7,48 @@
     <!-- Bootstrap core CSS -->
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" rel="stylesheet">
     <!-- Custom styles for this template -->
-    {#todo: <link href="custom.css" rel="stylesheet">#}
+
+    {# todo: move to assets! #}
+    <style>
+        body {
+            padding-top: 5rem;
+        }
+        .starter-template {
+            padding: 3rem 1.5rem;
+            text-align: center;
+        }
+        form label {
+            text-align: right;
+        }
+        .tab-pane {
+            margin-top: 20px;
+        }
+    </style>
 </head>
 <body>
-<nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
-    <a class="navbar-brand" href="#">Event Manager</a>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-    </button>
-    <div class="collapse navbar-collapse" id="navbarsExampleDefault">
-        <ul class="navbar-nav mr-auto">
-            <li class="nav-item">
-                <a class="nav-link" href="#">Events</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="{{ path('organizers') }}">Organizers</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="#">Languages</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="#">Mailing list</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="{{ path('admins') }}">Admins</a>
-            </li>
-        </ul>
-        <ul class="navbar-nav">
-            <li class="nav-item">
-                <a class="nav-link" href="{{ path('logout') }}">Logout</a>
-            </li>
-        </ul>
-    </div>
-</nav>
-<main role="main" class="container content">
-    <main role="main" class="container">
+    {% block navigation %}
+    {% endblock %}
+    <main role="main" class="container content">
```

Comment:
tomuto content-u som ja dala styly aby to nebolo nacapene navrchu. ale styly si zmazal. ak si to osetril nejak inak, tak zmaz aj tu triedu `content` inak jej styly nechaj.

Reply:
Osetril som to inak, tak ako to bolo povodne v prototype. Cize tuto classu mozeme zmazat.

Reply:
Dobre